scripts/upgrade

```diff
@@ -28,24 +42,35 @@ if [ -f "/etc/yunohost/apps/$app/scripts/backup" ] ; then
   }
 fi
 
-# move $data_path to new directory
-ynh_script_progression --message="Moving existing datas to new location..."
-if [ ! -d "$data_path" ] ; then
-  mv "/""home""/$app" "$data_path"
-  ynh_replace_string --match_string="/home/homeassistant/.homeassistant" --replace_string="$data_path/.$app" --target_file="$data_path/.$app/configuration.yaml"
-  chown -R $app: "$data_path"
-fi
-
-# grant sudo permissions to the user to manage his own systemd service
-ynh_script_progression --message="Creating dedicated user, rights and folders..."
-ynh_add_config --template="../conf/sudoers" --destination="/etc/sudoers.d/$app"
-
 # build (if needed) & install Pyhton
+ynh_script_progression --message="Installing dependencies..."
 myynh_install_dependencies --python="$PY_REQUIRED_VERSION"
 
 # stop systemd service
 ynh_script_progression --message="Stoping service..."
-ynh_systemd_action --service_name="$app@$app" --action=stop --line_match="Stopped Home Assistant" --log_path="/var/log/$app.log" --timeout=300
+ynh_systemd_action --service_name="$app" --action=stop --line_match="Stopped Home Assistant" --log_path="$log_file" --timeout=300
+
+# migrate to new app architecture
+ynh_script_progression --message="If needed, migrating to new app architecture..."
+if [ ! -d "$data_path" ]; then
+  # move $data_path to new directory
+  mv "/""home""/$app" "$data_path"
```

Comment:
```suggestion
  mv "/home/$app" "$data_path"
```

Reply:
to avoid linter detection that give a warning about /home but which is not relevant in these case